### ISSUE TITLE
[Bug report]: Tds-dropdown reset() method does not clear the filter value.

### ISSUE THREAD

**Issue by furkick** [scania-digital-design-system/tegel#766]
### Requirements before reporting

- [X] No duplicated [issue](https://github.com/scania-digital-design-system/tegel/issues) reported.
- [X] I have checked the latest version if the bug exist there. See all available packages at [npmJS.com](https://www.npmjs.com/package/@scania/tegel)
- [X] I have followed the installation guide.

### Package versions

@scania/tegel-angular-17: 1.13.0

### Browser

Chrome

### Framework

Angular

### Version

Angular 18

### Reproduction steps

1. Use code example
2. Type 'Option 2' in select to filter
3. Run the reset() method on the select.
4. The filter is not cleared.

### Code example

```
<tds-dropdown 
   name="dropdown" 
   label="Label text" 
   label-position="outside" 
   placeholder="Placeholder" 
   helper="Helper text" 
   size="lg" 
   open-direction="auto" 
   filter
   >
   <tds-dropdown-option value="option-1">Option 1</tds-dropdown-option>
   <tds-dropdown-option value="option-2">Option 2</tds-dropdown-option>
   <tds-dropdown-option value="option-3">Option 3</tds-dropdown-option>
</tds-dropdown>
```

### Expected behaviour

The reset() method should also clear the filter.

### Contact information

christopher.halfpenny@scania.com

**Comment by theJohnnyMe** [comment]
Fixed in tegel 1.17.0

**Comment by timrombergjakobsson** [comment]
[Merged here](https://github.com/scania-digital-design-system/tegel/pull/795)
